tests/Earthfile

```
@@ -256,6 +257,19 @@ copy-test-empty-src:
     DO +RUN_EARTHLY --earthfile=copy.earth --target=+copy-empty-src
     DO +RUN_EARTHLY --earthfile=copy.earth --target=+copy-empty-src-if-exists
 
+copy-test-wildcard:
+    DO +RUN_EARTHLY --earthfile=copy.earth --target=+copy-wildcard
+
+copy-test-if-exists-wildcard:
+    # Regression test for 'COPY --if-exists non_existent_folder/* /'
```

Nit: I would drop the "Regression test" from this comment, maybe just:
```suggestion
    # test 'COPY --if-exists non_existent_folder/* /' works when "non_existent_folder" doesn't exist.
```

---

I removed it as it's somewhat redundant anyhow.